quickstarts/gcp/gcp-redis/config.yml

```diff
@@ -0,0 +1,36 @@
+slug: gcp-redis
+description: |-
+  ## What is Redis?
+
+  Memorystore for Redis Cluster is a fully managed Redis service for Google Cloud. Applications running on Google Cloud can achieve extreme performance by leveraging the highly scalable, available, secure Redis service without the burden of managing complex Redis deployments.
+
+  ### Get started!
+
+  Start monitoring Redis by connecting Google Cloud Platform (GCP) to New Relic!
+  Check out our Redis documentation to instrument your cloud service and manage the stability, scalability, and reliability of your systems with 
+  New Relic's infrastructure monitoring capabilities.
+
+  ### More info
+
+  Check out the [documentation](https://docs.newrelic.com/docs/infrastructure/google-cloud-platform-integrations/gcp-integrations-list/google-memorystore-redis/) to learn more about New Relic monitoring for Redis.
+summary: Monitor Redis by connecting GCP to New Relic
+icon: logo.svg
+level: New Relic
+authors:
+  - New Relic
+title: Redis
```

Comment:
This quickstart already has the name Redis so this needs to be changed 
https://github.com/newrelic/newrelic-quickstarts/blob/a01923069e2662c87c5666c6dc175c48dc2a9f3b/quickstarts/redis/config.yml#L3